TravelPro.html

```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TravelPro</title>
+    <link rel="stylesheet" href="TravelPro.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+    <div class="navbar">
+        <div id="logo">
+            <img src="./images/logo.png" alt="logo" height="80px" width="110px">
+        </div>
+        <div id="options">
+            <a>Destinations</a>
+            <a>Hotels</a>
+            <a>Flights</a>
+            <a>Bookings</a>
+            <a>Login</a>
+            <a id="sign-up">Sign Up</a>
+            <select id="languages">
+                <option value="EN" selected><small>EN</small></option>
+                <option value="HIN"><small>HIN</small></option>
+                <option value="BN"><small>BN</small></option>
+            </select>
+        </div>
+    </div>
+    </header>
+
+
+
+    <div class="banner">
+        <div id="left">
+            <p id="sub_line">BEST DESTINATIONS AROUND THE WORLD</p>
+            <p id="banner_text">Travel, enjoy and live a new and full life</p>
+            <p id="sub_text">Built wicket longer admire do barton vanity itself do in it Preferred to sportsmen it engrossed listening. Park gate sell they west hard for the</p>
+            <button>Find out more</button>
+        </div>
+    </div>
+
+
+
+    <div class="services-heading">
+        <div>
+           <p>CATEGORY</p>
+           <p id="offer">We Offer Best Services</p>
+        </div>
+        <img src="./images/plus.png" height="160px" width="180px">
```

Comment:
Avoid inline styles such as height="80px" width="110px" on images. It's better to control these with CSS for maintainability and responsiveness.